internal/backend/sessions/sessioncalls/calls.go

```diff
@@ -63,12 +88,19 @@ func New(z *zap.Logger, config Config, svcs *sessionsvcs.Svcs) Calls {
 	return &cs
 }
 
-func (cs *calls) StartWorkers(ctx context.Context) error { return cs.worker.Start() }
+func (cs *calls) StartWorkers(ctx context.Context) error {
+	if err := cs.generalWorker.Start(); err != nil {
+		return err
+	}
+
+	return cs.uniqueWorker.Start()
+}
 
 func (cs *calls) Call(ctx workflow.Context, params *CallParams) (sdktypes.SessionCallAttemptResult, error) {
 	spec := params.CallSpec
 
 	fnv, _, _ := spec.Data()
```

Comment:
The diff is missing `	fnv, _, _ := spec.Data()` for some reason.

Reply:
?